cmd/deploy.go

```diff
@@ -292,3 +294,11 @@ func validImageRepo(image string) bool {
 	}
 	return result
 }
+
+func buildAstroUIDeploymentLink(deploymentName, wsID string) string {
+	context, err := config.GetCurrentContext()
+	if err != nil {
+		return ""
+	}
+	return fmt.Sprintf("https://app.%s/w/%s/d/%s", context.Domain, wsID, deploymentName)
```

Comment:
in some cases it can be http, pls use config https://github.com/astronomer/astro-cli/blob/301ea78fbdf7adecd799f894fcdd931bdd23dd15/config/config.go#L46

Reply:
Good catch 👍 , have updated the PR.